examples/TwoD_circle_restart.jl

```diff
@@ -0,0 +1,42 @@
+using WaterLily
+using ReadVTK, WriteVTK
+using StaticArrays
+
+function circle(p=4;Re=250,mem=Array,U=1)
+    # Define simulation size, geometry dimensions, viscosity
+    L=2^p
+    center,r = SA[3L,3L,0], L/2
+    # make a body
+    body = AutoBody() do xyz,t
+        x,y,z = xyz - center
+        √sum(abs2,SA[x,y,0])-r
+    end
+    Simulation((8L,6L,16),(U,0,0),L;ν=U*L/Re,body,mem)
+end
+
+# make a simulation on the CPU
+sim = circle();
+# make a vtk writer
+wr = vtkWriter("TwoD_circle_restart")
+# sim and then write and stop
+sim_step!(sim,1)
+write!(wr, sim)
+close(wr)
+
+# re-start the sim from a paraview file but on the GPU this time
+import CUDA
+@assert CUDA.functional()
```

Comment:
this is already asserted by the CUDAExt module when importing CUDA together with WaterLily. No need to call again here.
```suggestion
```